[ty] Add blurb for newer crates to `ty/CONTIBUTING.md (#22309)
astral-sh/ruff: crates/ty/CONTRIBUTING.md

    annotations for the Python standard library.
- `ty_wasm`: library crate for exposing ty as a WebAssembly module. Powers the
    [ty Playground](https://play.ty.dev/).

> - `ty_completion_eval`: framework for evaluating completion suggestions returned by the ty LSP.

Reviewer: ```suggestion
- `ty_completion_eval`: Framework for evaluating completion suggestions returned by the ty LSP.
```

> - `ty_module_resolver`: resolves Python modules across search paths allowing varying resolution modes.

Reviewer: ```suggestion
- `ty_module_resolver`: The module resolver, which allows resolving imports to their modules.
```

- `ty_static`: static data related to `ty`, currently only stores environment variable used by `ty`.

> - `ty_combine`: a utility crate containing the `Combine` trait.

Reviewer: ```suggestion
- `ty_combine`: Utility crate containing the `Combine` trait, which is used to combine `Options`.
```
